src/ts/Document.ts

```diff
@@ -52,8 +50,6 @@ class Document {
 	/** Indicates whether this document is a subdocument of a composing parent */
 	private readonly isSubdocument: boolean;
 
-	public constructor(schema: Schema, options: DocumentConstructorOptions);
```

Comment:
I had to undo these changes.  It turned out with how `Model` calls `super` here that it couldn't determine if there were appropriate parameters.